[INFO -- 2018-11-08 18:56:45]
Starting flip_value script
version: 1.1
[INFO -- 2018-11-08 18:56:45]
Initialization strings:
[INFO -- 2018-11-08 18:56:45]
(1) file /tmp/lreg/lreg
[INFO -- 2018-11-08 18:56:45]
(2) set args /tmp/lreg/randInput /tmp/lreg/output /tmp/lreg/detected.log
[INFO -- 2018-11-08 18:56:45]
initSignal:0
[INFO -- 2018-11-08 18:56:45]
endSignal:10
[INFO -- 2018-11-08 18:56:45]
faultModel:0
[EXCEPTION -- 2018-11-08 18:56:46]
pythonException: empty range for randrange() (0,0, 0)
[EXCEPTION -- 2018-11-08 18:56:46]
pythonException: Unable to exit loop in struct fields; Exiting wihtout making a bit flip
[INFO -- 2018-11-08 18:56:46]
Backtrace BEGIN:
#0  __GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:51
#1  0x00007ffff6ec6801 in __GI_abort () at abort.c:79
#2  0x00007ffff6f0f897 in __libc_message (action=action@entry=do_abort, fmt=fmt@entry=0x7ffff703cb9a "%s\n") at ../sysdeps/posix/libc_fatal.c:181
#3  0x00007ffff6f1690a in malloc_printerr (str=str@entry=0x7ffff703e870 "double free or corruption (out)") at malloc.c:5350
#4  0x00007ffff6f1de75 in _int_free (have_lock=0, p=0x5555557726c0, av=0x7ffff7271c40 <main_arena>) at malloc.c:4278
#5  __GI___libc_free (mem=0x5555557726d0) at malloc.c:3124
#6  0x0000555555557cc2 in __gnu_cxx::new_allocator<double>::deallocate (this=0x55555575b160 <y>, __p=0x5555557726d0) at /usr/include/c++/7/ext/new_allocator.h:125
#7  0x00005555555574d1 in std::allocator_traits<std::allocator<double> >::deallocate (__a=..., __p=0x5555557726d0, __n=16) at /usr/include/c++/7/bits/alloc_traits.h:462
#8  0x0000555555556b7c in std::_Vector_base<double, std::allocator<double> >::_M_deallocate (this=0x55555575b160 <y>, __p=0x5555557726d0, __n=16) at /usr/include/c++/7/bits/stl_vector.h:180
#9  0x0000555555556ffd in std::vector<double, std::allocator<double> >::_M_realloc_insert<double const&> (this=0x55555575b160 <y>, __position=0, __args#0=@0x7fffffffd690: 999) at /usr/include/c++/7/bits/vector.tcc:448
#10 0x0000555555556756 in std::vector<double, std::allocator<double> >::push_back (this=0x55555575b160 <y>, __x=@0x7fffffffd690: 999) at /usr/include/c++/7/bits/stl_vector.h:948
#11 0x00005555555559e5 in execute (input=0x7fffffffe1f1 "/tmp/lreg/randInput") at lreg.cpp:154
#12 0x0000555555555c60 in main (argc=4, argv=0x7fffffffde48) at lreg.cpp:191

46	../sysdeps/unix/sysv/linux/raise.c: No such file or directory.

Backtrace END
Fliping value of field: __val
Fliping array at pos: 5
Memory content before bitflip:0x000x000x000x000x000x000x000x00
Fault Model: Single bit-flip
base address to flip value: 0x7fffffffd148
address max offset: 8
Memory content after  bitflip:0x000x000x000x000x000x000x000x20
frame name: __GI_raise
symbol name: set
symbol filename: ../sysdeps/unix/sysv/linux/raise.c
symbol line: 39
value: {__val = {0, 0, 1, 8787296929185792, 999, 2305843009213693952, 1, 844420635172768, 140737488344560, 0, 140737488344596, 0, 4294967295, 140737351929539, 1, 0}}
value address: 0x7fffffffd120
Type: The type is a structure.
Type sizeof: 128
Field name: __val
Field Type: The type is an array.
Field Type sizeof: 128

[INFO -- 2018-11-08 18:56:46]
Fault Injection Successful after 1.9661338329315186s
[INFO -- 2018-11-08 18:56:47]
event type: exit
[EXCEPTION -- 2018-11-08 18:56:47]
exit code: no exit code available
